cam-dev.properties

```diff
@@ -0,0 +1,7 @@
+requestHandlerCodeURL=https://s3-ap-southeast-2.amazonaws.com/wps-lambda/request-handler-cam-dev-package.zip
```

Comment:
Perhaps add to your local .git/info/exclude file and remove from PR otherwise we'll end up with everyone's personal properties files in the repo

Reply:
Will remove